executing local tests in ./20210916/2/
executing local tests in ./20210916/1/
executing https://git.cs.msu.ru/s02190226/pythonprac/-/tree/main/20210916/1/tests on ./20210916/1/
executing https://github.com/gen-gematogen/pythonprac/tree/main/20210916/1/tests on ./20210916/1/
executing https://github.com/stamplevskiyd/pythonprac-2021/tree/main/20210916/1/tests on ./20210916/1/
executing local tests in ./20210916/3/
executing https://git.cs.msu.ru/s02190290/python-prac-2021/-/tree/main/20210916/3/tests on ./20210916/3/
executing https://git.cs.msu.ru/s02190700/pythonprac/-/tree/main/20210916/3/tests on ./20210916/3/
executing https://git.cs.msu.ru/s02180534/pythonprac2021/-/tree/main/20210916/3/tests on ./20210916/3/
executing local tests in ./20211014/2/
executing https://github.com/Uberariy/pythonprac/tree/main/20211014/2/tests on ./20211014/2/
executing https://git.cs.msu.ru/s02190290/python-prac-2021/-/tree/main/20211014/2/tests on ./20211014/2/
executing https://git.cs.msu.ru/s90190054/pythonprac-2021/-/tree/main/20211014/2/tests on ./20211014/2/
executing local tests in ./20211014/1/
executing https://github.com/OzerovaDaria/pythonprac/tree/main/20211014/1/tests on ./20211014/1/
executing https://github.com/gen-gematogen/pythonprac/tree/main/20211014/1/tests on ./20211014/1/
executing https://github.com/KH9IZ/PythonPrac/tree/main/20211014/1/tests on ./20211014/1/
executing local tests in ./20211014/3/
executing https://git.cs.msu.ru/s02190692/python-prac/-/tree/main/20211014/3/tests on ./20211014/3/
executing https://git.cs.msu.ru/s90190054/pythonprac-2021/-/tree/main/20211014/3/tests on ./20211014/3/
executing https://github.com/Veniamin-Arefev/pythonprac-2021/tree/main/20211014/3/tests on ./20211014/3/
executing local tests in ./20211111/2/
executing https://git.cs.msu.ru/s02190226/pythonprac/-/tree/main/20211111/2/tests on ./20211111/2/
executing https://git.cs.msu.ru/s02190029/prakpython/-/tree/main/20211111/2/tests on ./20211111/2/
executing https://github.com/sanyavertolet/pythonprac/tree/main/20211111/2/tests on ./20211111/2/
executing local tests in ./20211111/1/
executing https://git.cs.msu.ru/s02190248/pythonprac/-/tree/main/20211111/1/tests on ./20211111/1/
executing https://git.cs.msu.ru/s02190101/pythonprac-2021/-/tree/main/20211111/1/tests on ./20211111/1/
executing https://git.cs.msu.ru/s02190226/pythonprac/-/tree/main/20211111/1/tests on ./20211111/1/
executing local tests in ./20210923/2/
executing https://github.com/FluFFka/python-prac/tree/main/20210923/2/tests on ./20210923/2/
executing https://github.com/sanyavertolet/pythonprac/tree/main/20210923/2/tests on ./20210923/2/
executing https://git.cs.msu.ru/s02190248/pythonprac/-/tree/main/20210923/2/tests on ./20210923/2/
executing local tests in ./20210923/1/
executing https://git.cs.msu.ru/s90190054/pythonprac-2021/-/tree/main/20210923/1/tests on ./20210923/1/
executing https://github.com/FluFFka/python-prac/tree/main/20210923/1/tests on ./20210923/1/
executing https://git.cs.msu.ru/s02190141/pythonprac-2021/-/tree/main/20210923/1/tests on ./20210923/1/
executing local tests in ./20210923/3/
executing https://git.cs.msu.ru/s02190226/pythonprac/-/tree/main/20210923/3/tests on ./20210923/3/
executing https://git.cs.msu.ru/s02180534/pythonprac2021/-/tree/main/20210923/3/tests on ./20210923/3/
executing https://github.com/Veniamin-Arefev/pythonprac-2021/tree/main/20210923/3/tests on ./20210923/3/
executing local tests in ./20211125/2/
executing https://git.cs.msu.ru/s02190029/prakpython/-/tree/main/20211125/2/tests on ./20211125/2/
executing https://git.cs.msu.ru/s90190054/pythonprac-2021/-/tree/main/20211125/2/tests on ./20211125/2/
executing https://github.com/stamplevskiyd/pythonprac-2021/tree/main/20211125/2/tests on ./20211125/2/
executing local tests in ./20211125/1/
executing https://git.cs.msu.ru/s02190226/pythonprac/-/tree/main/20211125/1/tests on ./20211125/1/
executing https://git.cs.msu.ru/s02190248/pythonprac/-/tree/main/20211125/1/tests on ./20211125/1/
executing https://github.com/Uberariy/pythonprac/tree/main/20211125/1/tests on ./20211125/1/
executing local tests in ./20211125/3/
executing https://git.cs.msu.ru/s90190054/pythonprac-2021/-/tree/main/20211125/3/tests on ./20211125/3/
executing https://github.com/KH9IZ/PythonPrac/tree/main/20211125/3/tests on ./20211125/3/
executing https://git.cs.msu.ru/s02190290/python-prac-2021/-/tree/main/20211125/3/tests on ./20211125/3/
executing local tests in ./20211028/2/
executing https://git.cs.msu.ru/s02190226/pythonprac/-/tree/main/20211028/2/tests on ./20211028/2/
executing https://github.com/gen-gematogen/pythonprac/tree/main/20211028/2/tests on ./20211028/2/
executing https://git.cs.msu.ru/s02190029/prakpython/-/tree/main/20211028/2/tests on ./20211028/2/
executing local tests in ./20211028/1/
executing https://github.com/FluFFka/python-prac/tree/main/20211028/1/tests on ./20211028/1/
executing https://git.cs.msu.ru/s02190692/python-prac/-/tree/master/20211028/1/tests on ./20211028/1/
executing https://git.cs.msu.ru/s02190141/pythonprac-2021/-/tree/main/20211028/1/tests on ./20211028/1/
executing local tests in ./20211028/3/
executing https://github.com/KH9IZ/PythonPrac/tree/main/20211028/3/tests on ./20211028/3/
executing https://git.cs.msu.ru/s02190692/python-prac/-/tree/master/20211028/3/tests on ./20211028/3/
executing https://git.cs.msu.ru/s02190248/pythonprac/-/tree/main/20211028/3/tests on ./20211028/3/
executing local tests in ./20211118/2/
executing https://github.com/OzerovaDaria/pythonprac/tree/main/20211118/2/tests on ./20211118/2/
executing https://git.cs.msu.ru/s90190054/pythonprac-2021/-/tree/main/20211118/2/tests on ./20211118/2/
executing https://git.cs.msu.ru/s02190234/pythonprac2021/-/tree/main/20211118/2/tests on ./20211118/2/
executing local tests in ./20211118/1/
executing https://github.com/KH9IZ/PythonPrac/tree/main/20211118/1/tests on ./20211118/1/
executing https://git.cs.msu.ru/s02190226/pythonprac/-/tree/main/20211118/1/tests on ./20211118/1/
executing https://git.cs.msu.ru/s02190700/pythonprac/-/tree/main/20211118/1/tests on ./20211118/1/
executing local tests in ./20211118/3/
executing https://git.cs.msu.ru/s02180534/pythonprac2021/-/tree/main/20211118/3/tests on ./20211118/3/
executing https://github.com/Veniamin-Arefev/pythonprac-2021/tree/main/20211118/3/tests on ./20211118/3/
executing https://git.cs.msu.ru/s02190692/python-prac/-/tree/master/20211118/3/tests on ./20211118/3/
executing local tests in ./20211213/2/
executing local tests in ./20211213/1/
executing https://github.com/OzerovaDaria/pythonprac/tree/main/20211213/1/tests on ./20211213/1/
executing https://git.cs.msu.ru/s02190226/pythonprac/-/tree/main/20211213/1/tests on ./20211213/1/
executing https://git.cs.msu.ru/s02180534/pythonprac2021/-/tree/main/20211213/1/tests on ./20211213/1/
executing local tests in ./20211021/2/
executing https://github.com/Veniamin-Arefev/pythonprac-2021/tree/main/20211021/2/tests on ./20211021/2/
executing https://git.cs.msu.ru/s90190054/pythonprac-2021/-/tree/main/20211021/2/tests on ./20211021/2/
executing https://git.cs.msu.ru/s02190290/python-prac-2021/-/tree/main/20211021/2/tests on ./20211021/2/
executing local tests in ./20211021/1/
executing https://git.cs.msu.ru/s02190290/python-prac-2021/-/tree/main/20211021/1/tests on ./20211021/1/
executing https://git.cs.msu.ru/s02190029/prakpython/-/tree/main/20211021/1/tests on ./20211021/1/
executing https://git.cs.msu.ru/s02190700/pythonprac/-/tree/main/20211021/1/tests on ./20211021/1/
executing local tests in ./20211021/3/
executing https://git.cs.msu.ru/s02190226/pythonprac/-/tree/main/20211021/3/tests on ./20211021/3/
executing https://git.cs.msu.ru/s02190234/pythonprac2021/-/tree/main/20211021/3/tests on ./20211021/3/
executing https://git.cs.msu.ru/s02190700/pythonprac/-/tree/main/20211021/3/tests on ./20211021/3/
executing local tests in ./20211202/2/
executing https://git.cs.msu.ru/s02190692/python-prac/-/tree/master/20211202/2/tests on ./20211202/2/
executing https://git.cs.msu.ru/s02190141/pythonprac-2021/-/tree/main/20211202/2/tests on ./20211202/2/
executing https://git.cs.msu.ru/s02190101/pythonprac-2021/-/tree/main/20211202/2/tests on ./20211202/2/
executing local tests in ./20211202/1/
executing https://git.cs.msu.ru/s02190248/pythonprac/-/tree/main/20211202/1/tests on ./20211202/1/
executing https://github.com/gen-gematogen/pythonprac/tree/main/20211202/1/tests on ./20211202/1/
executing https://git.cs.msu.ru/s90190054/pythonprac-2021/-/tree/main/20211202/1/tests on ./20211202/1/
executing local tests in ./20211007/4/
executing https://git.cs.msu.ru/s02190029/prakpython/-/tree/main/20211007/4/tests on ./20211007/4/
executing https://github.com/sanyavertolet/pythonprac/tree/main/20211007/4/tests on ./20211007/4/
executing https://git.cs.msu.ru/s02190101/pythonprac-2021/-/tree/main/20211007/4/tests on ./20211007/4/
executing local tests in ./20211007/2/
executing https://github.com/stamplevskiyd/pythonprac-2021/tree/main/20211007/2/tests on ./20211007/2/
executing https://github.com/Veniamin-Arefev/pythonprac-2021/tree/main/20211007/2/tests on ./20211007/2/
executing https://git.cs.msu.ru/s02190226/pythonprac/-/tree/main/20211007/2/tests on ./20211007/2/
executing local tests in ./20211007/1/
executing https://git.cs.msu.ru/s02190700/pythonprac/-/tree/main/20211007/1/tests on ./20211007/1/
executing https://github.com/OzerovaDaria/pythonprac/tree/main/20211007/1/tests on ./20211007/1/
executing https://git.cs.msu.ru/s02190248/pythonprac/-/tree/main/20211007/1/tests on ./20211007/1/
executing local tests in ./20211007/3/
executing https://git.cs.msu.ru/s90190054/pythonprac-2021/-/tree/main/20211007/3/tests on ./20211007/3/
executing https://github.com/sanyavertolet/pythonprac/tree/main/20211007/3/tests on ./20211007/3/
executing https://git.cs.msu.ru/s02190700/pythonprac/-/tree/main/20211007/3/tests on ./20211007/3/
executing local tests in ./20210930/2/
executing https://github.com/FluFFka/python-prac/tree/main/20210930/2/tests on ./20210930/2/
executing https://git.cs.msu.ru/s02190290/python-prac-2021/-/tree/main/20210930/2/tests on ./20210930/2/
executing https://git.cs.msu.ru/s02190226/pythonprac/-/tree/main/20210930/2/tests on ./20210930/2/
executing local tests in ./20210930/1/
executing https://github.com/KH9IZ/PythonPrac/tree/main/20210930/1/tests on ./20210930/1/
executing https://git.cs.msu.ru/s02190692/python-prac/-/tree/master/20210930/1/tests on ./20210930/1/
executing https://github.com/FluFFka/python-prac/tree/main/20210930/1/tests on ./20210930/1/
executing local tests in ./20210930/3/
executing https://github.com/gen-gematogen/pythonprac/tree/main/20210930/3/tests on ./20210930/3/
executing https://git.cs.msu.ru/s02190226/pythonprac/-/tree/main/20210930/3/tests on ./20210930/3/
executing https://github.com/stamplevskiyd/pythonprac-2021/tree/main/20210930/3/tests on ./20210930/3/
################################################################################
python3 checkerNN.py ./20211014/2/ https://github.com/Uberariy/pythonprac/tree/main/20211014/2/tests
1.in: Output differs
--- 1.out
+++ <output>
@@ -1 +1 @@
-Size differs: 4221
+Size differs: <output>

2.in: Output differs
--- 2.out
+++ <output>
@@ -1 +1 @@
-Size differs: 8080
+Size differs: <output>

3.in: Output differs
--- 3.out
+++ <output>
@@ -1 +1 @@
-Size differs: 8040
+Size differs: <output>


################################################################################
python3 checkerNN.py ./20211014/2/ https://git.cs.msu.ru/s02190290/python-prac-2021/-/tree/main/20211014/2/tests
an error occured

################################################################################
python3 checkerNN.py ./20211014/2/ https://git.cs.msu.ru/s90190054/pythonprac-2021/-/tree/main/20211014/2/tests
1.in: Output differs
--- 1.out
+++ <output>
@@ -1 +1 @@
-Size differs: 820
+Size differs: <output>

2.in: Output differs
--- 2.out
+++ <output>
@@ -1 +1 @@
-Size differs: 1220
+Size differs: <output>

3.in: Output differs
--- 3.out
+++ <output>
@@ -1 +1 @@
-Size differs: 220
+Size differs: <output>


################################################################################
python3 checkerNN.py ./20211014/3/ https://git.cs.msu.ru/s02190692/python-prac/-/tree/main/20211014/3/tests
an error occured

################################################################################
python3 checkerNN.py ./20211014/3/ https://git.cs.msu.ru/s90190054/pythonprac-2021/-/tree/main/20211014/3/tests
1.in: Output differs
--- 1.out
+++ <output>
@@ -1 +1 @@
-Size differs: 102
+Size differs: <output>

2.in: Output differs
--- 2.out
+++ <output>
@@ -1 +1 @@
-Size differs: 114
+Size differs: <output>

3.in: Output differs
--- 3.out
+++ <output>
@@ -1 +1 @@
-Size differs: 90
+Size differs: <output>


################################################################################
python3 checkerNN.py ./20211014/3/ https://github.com/Veniamin-Arefev/pythonprac-2021/tree/main/20211014/3/tests
1.in: OK

2.in: OK

3.in: Output differs
--- 3.out
+++ <output>
@@ -1 +1 @@
-Size differs: 117
+Size differs: <output>


################################################################################
python3 checkerNN.py ./20211125/1/
1.in: Output differs
--- 1.out
+++ <output>
@@ -0,0 +1 @@
+qeeeeeeqqqqqrrrrrrwwwwww

2.in: OK

3.in: OK


################################################################################
python3 checkerNN.py ./20211125/1/ https://git.cs.msu.ru/s02190226/pythonprac/-/tree/main/20211125/1/tests
1.in: OK

2.in: Output differs
--- 2.out
+++ <output>
@@ -1 +1 @@
-Size differs: 101
+Size differs: <output>

3.in: Output differs
--- 3.out
+++ <output>
@@ -1 +1 @@
-Size differs: 101
+Size differs: <output>


################################################################################
python3 checkerNN.py ./20211028/3/ https://git.cs.msu.ru/s02190692/python-prac/-/tree/master/20211028/3/tests
1.in: Output differs
--- 1.out
+++ <output>
@@ -1 +1 @@
-Size differs: 359
+Size differs: <output>

2.in: OK

3.in: Output differs
--- 3.out
+++ <output>
@@ -1 +1 @@
-Size differs: 626
+Size differs: <output>


################################################################################
python3 checkerNN.py ./20211118/3/ https://git.cs.msu.ru/s02180534/pythonprac2021/-/tree/main/20211118/3/tests
1.in: OK

2.in: Output differs
--- 2.out
+++ <output>
@@ -1,2 +1,2 @@
 AttributeError in Alpha
-AttributeError in AlphaQ
+a: 1, b: 2, c: 3, d: 4

3.in: Output differs
--- 3.out
+++ <output>
@@ -1,2 +1,2 @@
 AttributeError in Alpha
-AttributeError in AlphaQ
+a: 1, b: 2, f: 5


################################################################################
python3 checkerNN.py ./20211118/3/ https://github.com/Veniamin-Arefev/pythonprac-2021/tree/main/20211118/3/tests
1.in: Output differs
--- 1.out
+++ <output>
@@ -1 +1 @@
-Size differs: 53
+Size differs: <output>

2.in: Output differs
--- 2.out
+++ <output>
@@ -1 +1 @@
-True
+False

3.in: Output differs
--- 3.out
+++ <output>
@@ -1 +1 @@
-True
+False


################################################################################
python3 checkerNN.py ./20211118/3/ https://git.cs.msu.ru/s02190692/python-prac/-/tree/master/20211118/3/tests
1.in: OK

2.in: Output differs
--- 2.out
+++ <output>
@@ -1 +1 @@
-Size differs: 167
+Size differs: <output>

3.in: Output differs
--- 3.out
+++ <output>
@@ -1 +1 @@
-Size differs: 119
+Size differs: <output>


################################################################################
python3 checkerNN.py ./20211213/1/ https://github.com/OzerovaDaria/pythonprac/tree/main/20211213/1/tests
0.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211213/1/prog.py", line 36, in <module>
    asyncio.run(joiner())
  File "/usr/lib/python3.8/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/home/ilysvtsky/pythonprac/20211213/1/prog.py", line 32, in joiner
    await events
TypeError: object dict can't be used in 'await' expression


1.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211213/1/prog.py", line 36, in <module>
    asyncio.run(joiner())
  File "/usr/lib/python3.8/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/home/ilysvtsky/pythonprac/20211213/1/prog.py", line 32, in joiner
    await events
TypeError: object dict can't be used in 'await' expression


2.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211213/1/prog.py", line 36, in <module>
    asyncio.run(joiner())
  File "/usr/lib/python3.8/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/home/ilysvtsky/pythonprac/20211213/1/prog.py", line 32, in joiner
    await events
TypeError: object dict can't be used in 'await' expression



################################################################################
python3 checkerNN.py ./20211213/1/ https://git.cs.msu.ru/s02190226/pythonprac/-/tree/main/20211213/1/tests
1.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211213/1/prog.py", line 36, in <module>
    asyncio.run(joiner())
  File "/usr/lib/python3.8/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/home/ilysvtsky/pythonprac/20211213/1/prog.py", line 32, in joiner
    await events
TypeError: object dict can't be used in 'await' expression


2.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211213/1/prog.py", line 36, in <module>
    asyncio.run(joiner())
  File "/usr/lib/python3.8/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/home/ilysvtsky/pythonprac/20211213/1/prog.py", line 32, in joiner
    await events
TypeError: object dict can't be used in 'await' expression


3.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211213/1/prog.py", line 36, in <module>
    asyncio.run(joiner())
  File "/usr/lib/python3.8/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/home/ilysvtsky/pythonprac/20211213/1/prog.py", line 32, in joiner
    await events
TypeError: object dict can't be used in 'await' expression



################################################################################
python3 checkerNN.py ./20211213/1/ https://git.cs.msu.ru/s02180534/pythonprac2021/-/tree/main/20211213/1/tests
1.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211213/1/prog.py", line 36, in <module>
    asyncio.run(joiner())
  File "/usr/lib/python3.8/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/home/ilysvtsky/pythonprac/20211213/1/prog.py", line 32, in joiner
    await events
TypeError: object dict can't be used in 'await' expression


2.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211213/1/prog.py", line 36, in <module>
    asyncio.run(joiner())
  File "/usr/lib/python3.8/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/home/ilysvtsky/pythonprac/20211213/1/prog.py", line 32, in joiner
    await events
TypeError: object dict can't be used in 'await' expression


3.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211213/1/prog.py", line 36, in <module>
    asyncio.run(joiner())
  File "/usr/lib/python3.8/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/home/ilysvtsky/pythonprac/20211213/1/prog.py", line 32, in joiner
    await events
TypeError: object dict can't be used in 'await' expression



################################################################################
python3 checkerNN.py ./20211021/2/ https://git.cs.msu.ru/s02190290/python-prac-2021/-/tree/main/20211021/2/tests
1.in: OK

2.in: Output differs
--- 2.out
+++ <output>
@@ -1,3 +1,4 @@
 2
 -1
-funcs:3\nstrs_numb:5
+funcs:3
+strs_numb:5

3.in: OK


################################################################################
python3 checkerNN.py ./20211021/3/ https://git.cs.msu.ru/s02190226/pythonprac/-/tree/main/20211021/3/tests
1.in: OK

2.in: OK

3.in: Output differs
--- 3.out
+++ <output>
@@ -0,0 +1 @@
+


################################################################################
python3 checkerNN.py ./20211021/3/ https://git.cs.msu.ru/s02190700/pythonprac/-/tree/main/20211021/3/tests
1.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211021/3/prog.py", line 4, in <module>
    while (s := input()):
EOFError: EOF when reading a line


2.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211021/3/prog.py", line 4, in <module>
    while (s := input()):
EOFError: EOF when reading a line


3.in: Output differs
--- 3.out
+++ <output>
@@ -1 +1 @@
-Size differs: 314
+Size differs: <output>


################################################################################
python3 checkerNN.py ./20211202/2/ https://git.cs.msu.ru/s02190692/python-prac/-/tree/master/20211202/2/tests
1.in: OK

2.in: OK

3.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211202/2/prog.py", line 17, in <module>
    exec(sys.stdin.read())
  File "<string>", line 9, in <module>
  File "/home/ilysvtsky/pythonprac/20211202/2/prog.py", line 7, in checker
    al = get_type_hints(self)
  File "/usr/lib/python3.8/typing.py", line 1264, in get_type_hints
    value = _eval_type(value, globalns, localns)
  File "/usr/lib/python3.8/typing.py", line 270, in _eval_type
    return t._evaluate(globalns, localns)
  File "/usr/lib/python3.8/typing.py", line 518, in _evaluate
    eval(self.__forward_code__, globalns, localns),
  File "<string>", line 1, in <module>
NameError: name 'myint' is not defined



################################################################################
python3 checkerNN.py ./20211202/1/ https://git.cs.msu.ru/s02190248/pythonprac/-/tree/main/20211202/1/tests
1.in: Output differs
--- 1.out
+++ <output>
@@ -1,5 +1,2 @@
-__init__: (10,), {}
-add: (9,), {}
 29
-add: (9,), {'another': 10}
 29

2.in: Output differs
--- 2.out
+++ <output>
@@ -1,5 +1,2 @@
-__init__: (10,), {}
-add: (9,), {}
 29
-add: (9,), {'another': 10}
 29

3.in: Output differs
--- 3.out
+++ <output>
@@ -1,5 +1,2 @@
-__init__: (10,), {}
-add: (9,), {}
 29
-add: (9,), {'another': 10}
 29


################################################################################
python3 checkerNN.py ./20211202/1/ https://github.com/gen-gematogen/pythonprac/tree/main/20211202/1/tests
1.in: Output differs
--- 1.out
+++ <output>
@@ -1,4 +1,2 @@
-__init__: ('Suzy',), {}
 Hello, my name is Suzy
-boo: (), {}
 Suzy is scared!

2.in: Output differs
--- 2.out
+++ <output>
@@ -1,4 +1,2 @@
-__init__: (3,), {}
 metametameta
-alien_mark: (1001,), {}
 maybe

3.in: Output differs
--- 3.out
+++ <output>
@@ -1,4 +1,2 @@
-__init__: (), {}
 Radio cheeeck!
-seb: (), {}
 Here comes Sebastion Vettel...


################################################################################
python3 checkerNN.py ./20211202/1/ https://git.cs.msu.ru/s90190054/pythonprac-2021/-/tree/main/20211202/1/tests
1.in: Output differs
--- 1.out
+++ <output>
@@ -1,5 +1,2 @@
-__init__: (10,), {}
-add: (9,), {}
 29
-add: (9,), {'another': 10}
 29

2.in: Output differs
--- 2.out
+++ <output>
@@ -1,6 +1,2 @@
-__init__: (4,), {}
-__init__: (2, 6), {}
-__str__: (), {}
 14
-__str__: (), {}
 8

3.in: Output differs
--- 3.out
+++ <output>
@@ -1,2 +1 @@
-methodd: (5,), {'e': 7, 'u': 78}
 90


################################################################################
python3 checkerNN.py ./20211007/4/ https://github.com/sanyavertolet/pythonprac/tree/main/20211007/4/tests
1.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211007/4/prog.py", line 13, in <module>
    frst, scnd, thrd = eval(input())
TypeError: cannot unpack non-iterable NoneType object


2.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211007/4/prog.py", line 13, in <module>
    frst, scnd, thrd = eval(input())
TypeError: cannot unpack non-iterable NoneType object


3.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211007/4/prog.py", line 13, in <module>
    frst, scnd, thrd = eval(input())
TypeError: cannot unpack non-iterable NoneType object



################################################################################
python3 checkerNN.py ./20211007/1/ https://git.cs.msu.ru/s02190700/pythonprac/-/tree/main/20211007/1/tests
1.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211007/1/prog.py", line 20, in <module>
    print(Pareto(*nums))
TypeError: Pareto() argument after * must be an iterable, not NoneType


2.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211007/1/prog.py", line 20, in <module>
    print(Pareto(*nums))
TypeError: Pareto() argument after * must be an iterable, not NoneType


3.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211007/1/prog.py", line 20, in <module>
    print(Pareto(*nums))
TypeError: Pareto() argument after * must be an iterable, not NoneType



################################################################################
python3 checkerNN.py ./20211007/3/ https://github.com/sanyavertolet/pythonprac/tree/main/20211007/3/tests
1.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211007/3/prog.py", line 19, in <module>
    print(Bisect(*eval(input())))
TypeError: Bisect() argument after * must be an iterable, not NoneType


2.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211007/3/prog.py", line 19, in <module>
    print(Bisect(*eval(input())))
TypeError: Bisect() argument after * must be an iterable, not NoneType


3.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211007/3/prog.py", line 19, in <module>
    print(Bisect(*eval(input())))
TypeError: Bisect() argument after * must be an iterable, not NoneType



################################################################################
python3 checkerNN.py ./20211007/3/ https://git.cs.msu.ru/s02190700/pythonprac/-/tree/main/20211007/3/tests
1.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211007/3/prog.py", line 19, in <module>
    print(Bisect(*eval(input())))
TypeError: Bisect() argument after * must be an iterable, not NoneType


2.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211007/3/prog.py", line 19, in <module>
    print(Bisect(*eval(input())))
TypeError: Bisect() argument after * must be an iterable, not NoneType


3.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20211007/3/prog.py", line 19, in <module>
    print(Bisect(*eval(input())))
TypeError: Bisect() argument after * must be an iterable, not NoneType



################################################################################
python3 checkerNN.py ./20210930/1/ https://git.cs.msu.ru/s02190692/python-prac/-/tree/master/20210930/1/tests
1.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20210930/1/prog.py", line 1, in <module>
    l = [int(x) for x in input().split(',')]
  File "/home/ilysvtsky/pythonprac/20210930/1/prog.py", line 1, in <listcomp>
    l = [int(x) for x in input().split(',')]
ValueError: invalid literal for int() with base 10: ''


2.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20210930/1/prog.py", line 1, in <module>
    l = [int(x) for x in input().split(',')]
  File "/home/ilysvtsky/pythonprac/20210930/1/prog.py", line 1, in <listcomp>
    l = [int(x) for x in input().split(',')]
ValueError: invalid literal for int() with base 10: ''


3.in: Traceback (most recent call last):
  File "/home/ilysvtsky/pythonprac/20210930/1/prog.py", line 1, in <module>
    l = [int(x) for x in input().split(',')]
  File "/home/ilysvtsky/pythonprac/20210930/1/prog.py", line 1, in <listcomp>
    l = [int(x) for x in input().split(',')]
ValueError: invalid literal for int() with base 10: ''



################################################################################
